src/cf.data.pre

```diff
@@ -3959,13 +3959,22 @@ DOC_START
 			configuration.
 
 	name=xxx	Unique name for the peer.
-			Required if you have multiple peers on the same host
-			but different ports.
-			This name can be used in cache_peer_access and similar
-			directives to identify the peer.
+			Required if you have multiple cache_peers with the same hostname.
+			Defaults to cache_peer hostname when not explicitly specified.
+
+			Other directives (e.g., cache_peer_access), cache manager reports,
+			and cache.log messages referring to a specific cache_peer use this
+			name to uniquely identify a cache_peer.
```

Comment:
```suggestion
			name to uniquely identify the cache_peer.
```

Reply:
FTR, speaking only for myself I do not insist. But to me "a X" implies vagueness about which X - whereas the subject of the sentence is about unique identification of one specific X.

Reply:
It's not a big deal for me either. However, what does the second 'cache_peer' stand for in this sentence then? Arbitrary (another) cache_peer or the same cache_peer that 'cache.log messages referring to'?

Reply:
This is not my area of expertise, but, AFAICT, the "logic" here is that we do not know what that "specific cache_peer" is in each of the described contexts (that peer depends on each of the described context details that are unknown to us), so we should not imply that we have somehow identified it or that the reader knows which specific cache_peer we are talking about. We did not, and they do not.

However, I may have found a different/better(?) way to phrase this: Instead of trying to describe how the `name` parameter is used in general, we can talk about a specific name value usage. See commit 6901173.

Reply:
That reads better to me, and its shorter :-)